src/modules/attitude_estimator_q/attitude_estimator_q_main.cpp

```diff
@@ -623,7 +623,13 @@ void AttitudeEstimatorQ::task_main()
 		ctrl_state.yaw_rate = _lp_yaw_rate.apply(_rates(2));
 
 		/* Airspeed - take airspeed measurement directly here as no wind is estimated */
-		ctrl_state.airspeed = _airspeed.indicated_airspeed_m_s;
+		if (PX4_ISFINITE(_airspeed.indicated_airspeed_m_s) && hrt_absolute_time() - _airspeed.timestamp < 1e6) {
```

Comment:
Can you add a "_airspeed.timestamp > 0 &&" here? Else it takes a second on boot where the airspeed is first valid.